crates/uv-distribution/src/error.rs

```diff
@@ -16,8 +16,6 @@ use uv_normalize::PackageName;
 pub enum Error {
     #[error("Building source distributions is disabled")]
     NoBuild,
```

Comment:
We do, however, need to be aggressive about rejecting _builds_ at metadata-fetch time.